apps/wallet-mobile/src/components/TokenIcon/TokenIcon.tsx

```diff
@@ -10,14 +10,27 @@ import {Boundary} from '../Boundary'
 import {Icon} from '../Icon'
 import {ModeratedNftIcon} from './ModeratedNftIcon'
 
-export const TokenIcon = ({wallet, tokenId}: {wallet: YoroiWallet; tokenId: string}) => {
+export const TokenIcon = ({
+  wallet,
+  tokenId,
+  size,
+}: {
+  wallet: YoroiWallet
+  tokenId: string
+  size?: 'small' | 'medium' | 'large'
```

Comment:
it looks only `small` and something else was implemented is that right?

Reply:
yes! only `small` is implemented in the component as I have left the previous styles for it as default in case the user is not providing any prop to `size`